src/@next/Card/CardStyle.ts

```diff
@@ -34,18 +34,38 @@ export const StyledCardHeaderSectionHalf = styled.div`
 `;
 
 export const StyledCardContentWrapper = styled.div`
-  padding-left: ${space4};
+  padding-bottom: ${space4};
+
+  @media (max-width: ${Breakpoints.large}) {
+    padding-bottom: 0px;
```

Comment:
For smaller screen, there is no need to add extra padding-bottom since the Section itself has padding-bottom = Header padding-top. So it's even for the overall padding top-bottom of the card